src/components/enterprise-banner/tests/EnterpriseBanner.test.jsx

```diff
@@ -21,6 +21,7 @@ describe('<EnterpriseBanner />', () => {
 
     render(
       <AppContext.Provider
+        // eslint-disable-next-line react/jsx-no-constructed-context-values
```

Comment:
Similar question about being consistent on how we're handling this ESLint rule (in some test files, you're using useMemo, but in others like here, you're not).